guides/common/modules/con_postgresql-as-an-external-database-considerations.adoc

```diff
@@ -10,7 +10,13 @@ ifndef::katello,orcharhino,satellite[]
 Foreman uses the PostgreSQL database.
 endif::[]
 If you want to use PostgreSQL as an external database, the following information can help you decide if this option is right for your {Project} configuration.
+ifdef::foreman-deb[]
+{Team} tests {Project} with PostgreSQL version 13.
```

Comment:
For `foreman-deb` we don't test with PostgreSQL 13 because we only test with the distribution default versions (so 14 & 15).

For EL9 we also only use the distro default version so for upstream I think you can say we require version 13 (because that's what Pulp requires) and recommend the distribution default versions.